frontend/packages/helm-plugin/src/utils/helm-detection-utils.ts

```diff
@@ -14,17 +15,20 @@ const checkDisabledHelmCharts = (helmChartRepositories: K8sResourceKind[]): bool
 
 export const detectHelmChartRepositories = async (setFeatureFlag: SetFeatureFlag) => {
   let id = null;
-  const fetchHelmChartRepositories = () =>
-    fetchK8s<ListKind<K8sResourceKind>>(HelmChartRepositoryModel)
-      .then((list) => {
-        setFeatureFlag(FLAG_OPENSHIFT_HELM, !checkDisabledHelmCharts(list?.items));
-      })
-      .catch((error) => {
-        error?.response?.status === 404
-          ? setFeatureFlag(FLAG_OPENSHIFT_HELM, false)
-          : setFeatureFlag(FLAG_OPENSHIFT_HELM, undefined);
-        clearInterval(id);
-      });
-  fetchHelmChartRepositories();
-  id = setInterval(fetchHelmChartRepositories, 10 * 1000);
+  if (isMultiClusterEnabled()) setFeatureFlag(FLAG_OPENSHIFT_HELM, false);
```

Comment:
nit, but I think this get merged over night :P

```suggestion
  if (isMultiClusterEnabled()) {
    setFeatureFlag(FLAG_OPENSHIFT_HELM, false);
    return;
  }
```